src/acpustatus.cc

```diff
@@ -252,21 +252,44 @@ void CPUStatus::draw(Graphics& g) {
 }
 
 void CPUStatus::temperature(Graphics& g) {
-    if (cpustatusShowAcpiTempInGraph) {
-        char temp[10];
-        getAcpiTemp(temp, sizeof(temp));
-        g.setColor(fTempColor);
-        if (tempFont == null)
-            tempFont = tempFontName;
-        if (tempFont) {
-            g.setFont(tempFont);
-            int h = height();
-            int y = (h - 1 - tempFont->height()) / 2 + tempFont->ascent();
```

Comment:
I am not sure why there was a - 1 here, but I removed it because I thought it looked more centered without it.